apps/directory/src/views/ConfigurationScreen.vue

```diff
@@ -0,0 +1,380 @@
+  adaptive: false                                            /** if the filters options should react on search results                                         */
+  sortColumn: "name"                                         /** specify a column to apply sorting on                                                          */
+  sortDirection: "asc"                                       /** the direction to sort                                                                         */
+}
+          </pre>
+        </small>
+      </div>
+    </div>
+
+    <!-- Advanced Editor -->
+    <json-editor
+      :config="currentConfig"
+      @save="saveFromEditor"
+      v-if="editorType === 'editor'"
+    />
+
+    <!-- Diff editor -->
+    <diff-editor
+      v-if="editorType === 'diff'"
+      :currentConfig="currentConfig"
+      :newConfig="uploadedAppConfig"
+      @save="saveFromEditor"
+      @cancel="switchView('editor')"
+    />
+
+    <!-- Landingpage editor -->
+    <div v-if="editorType === 'landingpage'" class="row px-5 pb-5">
+      <landingpage-editor
+        :currentConfig="currentConfig"
+        @save="saveLandingpage"
+        @cancel="switchView('editor')"
+      />
+    </div>
+    <!-- standard button bar -->
+    <div v-if="currentView === 'ui'" class="row px-5 pb-5">
+      <div class="col pl-0">
+        <button class="btn btn-primary mr-3 save-button" @click="save">
+          Save configuration
+        </button>
+        <button v-if="dirty" class="btn btn-dark mr-3" @click="cancel">
+          Cancel
+        </button>
+
+        <button class="btn btn-outline-dark mr-3" @click="download">
+          Download config
+        </button>
+        <button class="btn btn-outline-dark" @click="upload">
+          Upload config
+        </button>
+
+        <input
+          type="file"
+          id="file-selector"
+          accept=".json"
+          @change="processUpload"
+        />
+      </div>
+      <div>
+        <div class="row">
+          <div v-show="showNotification">
+            <div
+              v-if="configUpdateStatus === 204"
+              class="alert alert-success m-0 mr-3"
+              role="alert"
+              @click="statusClosed = true"
+            >
+              <span>Configuration saved!</span>
+            </div>
+            <div
+              v-else
+              class="alert alert-warning m-0"
+              role="alert"
+              @click="statusClosed = true"
+            >
+              <span
+                >We could not save the configuration, make sure you are logged
+                in with sufficient rights.</span
+              >
+            </div>
+          </div>
+
+          <div v-show="dirty" class="alert alert-warning m-0" role="alert">
+            <span>You have unsaved changes</span>
+          </div>
+        </div>
+        <small class="mt-4 float-right">To format your file press ctrl + f</small>
```

Comment:
I think this only needs to be shown when the editor is also shown.

Reply:
done!